src/model/CustomerInformation.ts

```diff
@@ -0,0 +1,79 @@
+import { Type } from 'class-transformer';
+import { CustomDataType } from './CustomData';
+import { IdTokenProps, IdToken } from '../pages/id-tokens/IdToken';
+import { GqlAssociation } from '../util/decorators/GqlAssociation';
+import { ChargingStation } from '../pages/charging-stations/ChargingStation';
+import { ADDITIONAL_INFOS_RELATED_IDTOKENS } from '../queries/additionalInfo';
+import {
+  IsString,
+  IsOptional,
+  ValidateNested,
+  IsInt,
+  IsPositive,
+  IsNotEmpty,
+  IsBoolean,
+  MinLength,
+} from 'class-validator';
+
+const ID_TOKEN_FIELD = 'idToken';
+
+export class GetCustomersRequest {
```

Comment:
very minor, is there a reason this request is called `GetCustomerRequest` rather than `CustomerInformationRequest` ? since the OCPP docs outline this particular request as a `CustomerInformationRequest`, i wonder if it would add more clarity if this request was named the same way.

Reply:
Totally aligned